docs/src/pages/getting-started/installation/troubleshooting.react.mdx

```diff
@@ -90,3 +90,26 @@ export default defineConfig({
     "skipLibCheck": true,
 ...
 ```
+
+### Jest
+
+In latest versions of `@aws-amplify/ui-react`, users of the Jest testing framework may run into the following error
```

Comment:
Should we state the specific version this started in and up? e.g. `in versions of `@aws-amplify/ui-react` 2.15.3 and above` (not sure if that is the correct version we introduced `MapView` in btw 😉)

Reply:
Yes that's much better.